yarn-project/pxe/src/contract_function_simulator/oracle/private_execution_oracle.ts

```diff
@@ -40,8 +40,9 @@ import { UtilityExecutionOracle } from './utility_execution_oracle.js';
 export class PrivateExecutionOracle extends UtilityExecutionOracle {
   /**
    * New notes created during this execution.
-   * It's possible that a note in this list has been nullified (in the same or other executions) and doesn't exist in the ExecutionNoteCache and the final proof data.
-   * But we still include those notes in the execution result because their commitments are still in the public inputs of this execution.
+   * It's possible that a note in this list has been nullified (in the same or other executions) and doesn't exist in
+   * the ExecutionNoteCache and the final proof data. But we still include those notes in the execution result because
+   * their commitments are still in the public inputs of this execution.
```

Comment:
Unrelated change --> 120 char line width was not respected and it made me unhappy